userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -776,29 +828,29 @@ image:PaymentMethodStar-Labeled.png[width=350,align="center"]
 
 ==== Applying Transactions to a Payment Method
 
-This section explains how to apply a transaction to a payment method. These transactions include:
+This section explains how to apply a transaction to a payment method. These transactions are directly applied on the payment instrument (as opposed to being applied to the unpaid invoice).
+
+You should only process transactions from the Payment Methods area if the transaction is unrelated to an invoice and/or payment. For example, to refund a payment associated with a specific invoice, you should instead <<Refund a Payment, refund the payment>> from the Payment Details page.
+
+These transactions include:

```

Comment:
It might be useful to add some context here. For example, under what situation would an operator need to apply such transactions? I'm not familiar with applying transactions, so got me wondering, so other users might also have similar questions. But if it's too technical or complicated, we can leave it out for now.

Reply:
Maybe point to our [payment manual](https://docs.killbill.io/latest/userguide_payment.html) ?

Reply:
I tried to give it a bit more context. Look for the next version of this.